markdown/developers/guidelines/requirements/git_branches.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Use nf-core git branches
+subtitle: Use `master`, `dev` and `TEMPLATE`.
+menu:
+  main:
+    weight: 160
+---
+
+The latest stable release should be on the main `master` branch.
+No additional changes should be pushed to master after each release.
+
+The main development code should be kept in a branch called `dev`.
+The nf-core `dev` branch should be set as the default branch up until the first release.
```

Comment:
Should we mention patch branches (I believe these are stll in the template, and I find that useful - particularly now trying to port to DSL2)? 

Speaking of which - do we mention about numbers of reviews? And the importance of PRs/reviews anywhere? IIRC this was in the old guidelines, or id you remove this on purpose?

Reply:
```suggestion
The nf-core `dev` branch should be set as the default branch up until the first release.

For minor bugfixes a `patch` branch may be used and merged directly into `master`, leaving `dev` for continued development work.
```

Reply:
I don't think that reviews were ever in the guidelines. They should be handled by branch protection though, so not sure that we need to mention them here.